modules/RTC/JitsiLocalTrack.js

```diff
@@ -112,8 +112,8 @@ export default class JitsiLocalTrack extends JitsiTrack {
         // Safari returns an empty constraints object, construct the constraints using getSettings.
         if (!Object.keys(this._constraints).length && videoType === VideoType.CAMERA) {
             this._constraints = {
-                height: track.getHeight(),
-                width: track.getWidth()
+                height: track.getSettings().getHeight(),
```

Comment:
Why not this.getHeight() ?

Reply:
I think I can help: track here is MediaStreamTrack, not jitsi track, which does not have `getHeight` and that caused issue https://github.com/jitsi/lib-jitsi-meet/issues/2379 

I would probably suggest checking if it is possible to rename variable as nativeTrack or something to prevent further confusion

Reply:
oh, I see your point, sorry, missed `this` instead of `track`. Good point though!

Reply:
no particular reason for this.getHeight()

Reply:
I'll replace it